apps/src/applab/designElements/library.js

```diff
@@ -122,12 +124,15 @@ export default {
    */
   getElementType: function(element, allowUnknown) {
     var tagname = element.tagName.toLowerCase();
-
     switch (tagname) {
       case 'button':
         return ElementType.BUTTON;
       case 'label':
```

Comment:
Question: Why make this a label instead of a button?

Reply:
I'm using a label to hide the ugly file upload input element. I couldn't find an easy way to make a button do the file upload so I'm using a label to make the photo chooser look and act like a button but actually trigger the input element offscreen.